apps/mysql.go

```diff
@@ -55,70 +57,137 @@ func (r MetricsReceiverMySql) Pipelines() []otel.ReceiverPipeline {
 		r.Username = "root"
 	}
 
-	return []otel.ReceiverPipeline{{
-		Receiver: otel.Component{
-			Type: "mysql",
-			Config: map[string]interface{}{
-				"collection_interval": r.CollectionIntervalString(),
-				"endpoint":            r.Endpoint,
-				"username":            r.Username,
-				"password":            r.Password,
-				"transport":           transport,
-				"metrics": map[string]interface{}{
-					"mysql.commands": map[string]interface{}{
-						"enabled": true,
-					},
-					"mysql.index.io.wait.count": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.index.io.wait.time": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.locked_connects": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.mysqlx_connections": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.opened_resources": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.tmp_resources": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.prepared_statements": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.table.io.wait.count": map[string]interface{}{
-						"enabled": false,
-					},
-					"mysql.table.io.wait.time": map[string]interface{}{
-						"enabled": false,
-					},
+	// MySQL replication metrics are implemented separate to the main metrics pipeline so that 5.7 and 8.0 are both supported,
+	// this configuration is used for the sqlqueryreceiver
```

Comment:
Nit: the term `sqlqueryreceiver` no longer appears anywhere, right? You could add a space to turn it into "sqlquery receiver" (appearing below), or avoid the need for this comment altogether by renaming this variable, e.g., to `sqlqueryReceiverDriverConf` or even `sqlReceiverDriverConfig` (to match `sqlReceiverQueriesConfig`).

Reply:
Resolved.